pkg/apis/configuration/v1/types.go

```diff
@@ -180,6 +190,20 @@ type TLSRedirect struct {
 	BasedOn string `json:"basedOn"`
 }
 
+// VirtualServerStatus defines the status for the VirtualServer resource.
+type VirtualServerStatus struct {
+	State             string             `json:"state"`
+	Reason            string             `json:"reason"`
+	Message           string             `json:"message"`
+	ExternalEndpoints []ExternalEndpoint `json:"externalEndpoints,omitempty"`
+}
+
+// ExternalEndpoint defines the IP/Hostname and ports used to connect to this resource.
+type ExternalEndpoint struct {
```

Comment:
this got outdated because of the previous changes.
```suggestion
// ExternalEndpoint defines the IP and ports used to connect to this resource.
```

please make sure to update generated CRDs for the helm chart as well.